oct/cli/package/common_options.py

```diff
@@ -14,21 +14,29 @@ def package_options(func):
     :return: decorated CLI command
     """
     return option(
-        '--update/--upgrade',
-        '-d/-g',
-        'update_current_stage',
-        default=False,
-        help='Update stage or upgrade to next stage.  [default: upgrade]',
+        '--upgrade',
```

Comment:
The semantics of this are a little complicated to understand. Do you think:

```
oct package ami --strategy update
oct package ami --strategy upgrade
oct package ami --strategy stage=<stage>
```

Is simpler?